nautobot/dcim/views.py

```diff
@@ -2873,6 +2874,9 @@ def get_extra_context(self, request, instance):
         vlan_table = InterfaceVLANTable(interface=instance, data=vlans, orderable=False)
 
         redundancy_table = self._get_interface_redundancy_groups_table(request, instance)
+        virtual_device_contexts_table = tables.VirtualDeviceContextTable(
+            instance.virtual_device_contexts.all(), orderable=False, exclude=("device",)
```

Comment:
Any select_related here? as well as 
```suggestion
            instance.virtual_device_contexts.restrict(request.user, "view"), orderable=False, exclude=("device",)
```